README.md

```diff
@@ -74,7 +74,7 @@ Here's a more full-fledged example, still using `#kaocha {}`.
 
           ;; Regex patterns to determine whether a namespace contains
           ;; tests.
-          :ns-patterns [#"-test$"]}]
```

Comment:
with the leading `#` I get the following exception; and removing it makes the error go away
```
clojure -A:test --print-config
Exception in thread "main" clojure.lang.ExceptionInfo: Config error on line 18 {:line 18}
	at aero.core$read_config_into_tag_wrapper.invokeStatic(core.cljc:347)
	at aero.core$read_config_into_tag_wrapper.invoke(core.cljc:329)
	at aero.core$read_config.invokeStatic(core.cljc:375)
	at aero.core$read_config.invoke(core.cljc:362)
	at aero.core$read_config.invokeStatic(core.cljc:380)
	at aero.core$read_config.invoke(core.cljc:362)
	at kaocha.config$load_config.invokeStatic(config.clj:51)
	at kaocha.config$load_config.invoke(config.clj:45)
	at kaocha.runner$_main_STAR_.invokeStatic(runner.clj:70)
	at kaocha.runner$_main_STAR_.doInvoke(runner.clj:68)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:657)
	at clojure.core$apply.invoke(core.clj:652)
	at kaocha.runner$_main.invokeStatic(runner.clj:135)
	at kaocha.runner$_main.doInvoke(runner.clj:133)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.core$apply.invokeStatic(core.clj:657)
	at clojure.main$main_opt.invokeStatic(main.clj:378)
	at clojure.main$main_opt.invoke(main.clj:374)
	at clojure.main$main.invokeStatic(main.clj:485)
	at clojure.main$main.doInvoke(main.clj:448)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:705)
	at clojure.main.main(main.java:37)
Caused by: clojure.lang.EdnReader$ReaderException: java.lang.RuntimeException: No dispatch macro for: "
	at clojure.lang.EdnReader.read(EdnReader.java:180)
	at clojure.lang.EdnReader.read(EdnReader.java:111)
	at clojure.edn$read.invokeStatic(edn.clj:35)
	at clojure.edn$read.invoke(edn.clj:14)
	at aero.core$read_config_into_tag_wrapper.invokeStatic(core.cljc:335)
	... 24 more
Caused by: java.lang.RuntimeException: No dispatch macro for: "
	at clojure.lang.Util.runtimeException(Util.java:221)
	at clojure.lang.EdnReader$DispatchReader.invoke(EdnReader.java:552)
	at clojure.lang.EdnReader.readDelimitedList(EdnReader.java:757)
	at clojure.lang.EdnReader$VectorReader.invoke(EdnReader.java:672)
	at clojure.lang.EdnReader.readDelimitedList(EdnReader.java:757)
	at clojure.lang.EdnReader$MapReader.invoke(EdnReader.java:680)
	at clojure.lang.EdnReader.readDelimitedList(EdnReader.java:757)
	at clojure.lang.EdnReader$VectorReader.invoke(EdnReader.java:672)
	at clojure.lang.EdnReader.readDelimitedList(EdnReader.java:757)
	at clojure.lang.EdnReader$MapReader.invoke(EdnReader.java:680)
	at clojure.lang.EdnReader.read(EdnReader.java:145)
	at clojure.lang.EdnReader$TaggedReader.readTagged(EdnReader.java:790)
	at clojure.lang.EdnReader$TaggedReader.invoke(EdnReader.java:783)
	at clojure.lang.EdnReader$DispatchReader.invoke(EdnReader.java:549)
	at clojure.lang.EdnReader.read(EdnReader.java:145)
```